evm/utils.js

```diff
@@ -407,7 +407,7 @@ const getEVMAddresses = async (config, chain, options = {}) => {
     const keyID = options.keyID || '';
```

Comment:
`keyID` from CLI is being used here so shouldn't remove the option above

Reply:
added the option again